plugin.json

```
@@ -2,6 +2,10 @@
     "id": "com.mattermost.plugin-incident-response",
     "name": "Incident Response",
     "description": "This plugin allows users to coordinate and manage incidents within Mattermost.",
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-incident-response/",
+    "support_url": "https://github.com/mattermost/mattermost-plugin-incident-response//issues",
```

```suggestion
    "support_url": "https://github.com/mattermost/mattermost-plugin-incident-response/issues",
```
(typo on the last slash)

---

Good catch :+1: